src/imitation/scripts/parallel.py

```diff
@@ -47,15 +49,28 @@ def parallel(
       local_dir: `local_dir` argument to `ray.tune.run()`.
       upload_dir: `upload_dir` argument to `ray.tune.run()`.
     """
+    # Basic validation for config options before we enter parallel jobs.
+    for name in base_named_configs:
+        assert isinstance(name, str)
+    for k in base_config_updates:
+        assert isinstance(k, str)
+    assert isinstance(search_space["named_configs"], collections.abc.Sequence)
+    assert isinstance(search_space["config_updates"], collections.abc.Mapping)
+
     # Explicitly set `data_dir` if parallelizing `train_adversarial`.
-    # We need this to automatically find rollout pickles Ray will automatically
-    # change the working directory for each Raylet.
+    # We need this to automatically find rollout pickles because Ray
+    # sets a new working directory for each Raylet.
     if sacred_ex_name == "train_adversarial":
         if "data_dir" not in base_config_updates:
-            base_config_updates["data_dir"] = os.path.join(os.getcwd(), "data/")
+            data_dir = os.path.join(os.getcwd(), "data/")
+            base_config_update = dict(base_config_updates)
```

Comment:
Typo? `base_config_update` != `base_config_updates`. If these are meant to be different then please rename to something more clearly distinct.

Reply:
:+1: this is a typo.